### DISCUSSION TITLE
Confusion between ctfeatures and unassertable higher geography

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#8045] [General]
In the geography committee meeting today, it was clear that we are using "features" to describe several things. One of these were dubbed "Fun Flexible Features" or "FFFs" when we made the transition to spatial higher geography. FFFs are polygons for geographic areas that can be used in search, but are not "assertable" (they cannot be assigned as higher geography).

The documentation refers to these as [Unassertable Higher Geography](https://handbook.arctosdb.org/documentation/higher-geography.html#unassertable-higher-geography)

I propose that we select an appropriate name for these  so that they are not confused with the locality attribute "feature", which is a string value that can be assigned to localities. I'll throw down a few, but suggestions are appreciated!

spatial features
geographic features
search polygons
spatial search

We need to be able to request new "FFFs" and knowing what to ask for will be a necessity, even if we cannot see the list of available FFFs, however, I think we should be able to see that so that we can make good decisions about requests for new polygons. So I also request that this list be visible somewhere. 

@ArctosDB/geo-group

**Comment by genevieve-anderegg** [comment]
I like "geographic features", because these shapes are usually an area that has a distinct name or "idea" associated with them. "Search polygons" is also good because that describes precisely what they are and how they are used in Arctos. I'm sure someone will have a better and more specific than I though.

>  I think we should be able to see that so that we can make good decisions about requests for new polygons. So I also request that this list be visible somewhere.

Yes please!!! 

Thank you so much for filing this issues after the meeting Teresa! 💪

**Comment by dustymc** [comment]
> FFFs are 

unassertable geography - they can carry all the information any geography record can.

>spatial features

They're spatial, but so is (almost) everything else.

> geographic features

Ditto - they are, but that's not unique.

> search polygons

They're for search, but they're not just polygons

> spatial search

FFFs aren't the only available polygons, all can be used for search.


> "geographic features", 

But not this one:

```
arctosprod@arctos>> \d geog_auth_rec
                                                           Table "core.geog_auth_rec"
          Column          |          Type           | Collation | Nullable |                              Default                               
--------------------------+-------------------------+-----------+----------+--------------------------------------------------------------------
 geog_auth_rec_id         | integer                 |           | not null | 
 ...
 feature                  | character varying(50)   |           |          | 

```

I'm not sure any of that's fatal, I sure don't have better ideas, but none of it really grabs me either....

**Comment by genevieve-anderegg** [comment]
I keep thinking the word "landscape" may be useful here. Searchable landscape features?
